jenkins/bootstrap.py

```diff
@@ -941,8 +941,13 @@ def bootstrap(args):
     build = build_name(started)
 
     if upload:
-        if any(repos[repo][1] for repo in repos):
+        # TODO(bentheelder, cjwager, stevekuznetsov): support the workspace
```

Comment:
cc @stevekuznetsov @cjwagner 
we have a few jobs that do this now, generally they work from k8s but integrate against a PR in another repo.